File: src/test/java/com/github/jactor/junit/GreetJUnitJupiterTest.java
package com.github.jactor.junit;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

class GreetJUnitJupiterTest {

    @Test
    @DisplayName("should greet the world")
    void shouldGreetTheWorld() {
        assertThat(new Greet().sayHello()).isEqualTo("Hello World!");
    }

    @Test
    @DisplayName("should greet a person")
    void shouldGreetPerson() {
        assertThat(new Greet("Tor Egil").sayHello()).isEqualTo("Hello Tor Egil!");
    }
}

File: src/test/kotlin/com/github/jactor/junit/GreetJUnitJupiterTest.kt
package com.github.jactor.junit

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class GreetJUnitJupiterTest {

    @Test
    fun `should greet the world`() {
        assertThat(Greet().sayHello()).isEqualTo("Hello World!")
    }

    @Test
    fun `should greet a person`() {
        assertThat(Greet("Tor Egil").sayHello()).isEqualTo("Hello Tor Egil!")
    }
}